pemFioi/conceptViewer-1.0-mobileFirst.js

```diff
@@ -254,7 +254,8 @@ var conceptViewer = {
   },
 
   openInNewWidget: function() {
-    var url = "https://static4.castor-informatique.fr/help/display-documentation.html";
+    // we use the function to get the base url in order to support http and https.
+    var url = getConceptViewerBaseUrl().replace("index.html", "display-documentation.html");
```

Comment:
getConceptViewerBaseUrl doesn't always return an URL containing `index.html`, sometimes it can be `index_xx.html` in which case your replace won't work.

I would suggest modifying getConceptViewerBaseUrl, with two possibilities :

* (preferred) either just remove the end of the URL, and move the 5 lines related to index.html to getConceptViewerBaseConcepts, while adding display-documentation.html to the URL in openInNewWidget
* or have a parameter which tells whether it should generate the URL to index or to display-documentation

Reply:
Ha yes, I forgot that the getConceptBaseUrl create url depending on the language.

I have fixed that with the prefered method in the next commit